scripts/easy_train.py

```
@@ -1647,16 +1647,18 @@ def prepare_start_model_from_experiment(directory, experiment_path, run_id, nnue
         raise Exception('Could not find any viable .ckpt nor .nnue files in the start experiment.')
     return prepare_start_model(directory, best_model, run_id, nnue_pytorch_directory, features)
 
+def get_default_feature_set_from_nnue_pytorch(nnue_pytorch_directory):
+    with open(os.path.join(nnue_pytorch_directory, 'features.py'), 'r') as features_file:
+        for line in features_file:
+            line = line.strip()
+            if line.startswith('_default_feature_set_name'):
+                return line.split()[-1][1:-1]
+
```

should we throw an exception if nothing found?